src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,62 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_ELEMENTS_LENGTH = 10;
+    private static final int NOT_FOUND_ELEM_IDX = -1;
+
+    private final K[] keys;
+    private final V[] values;
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```
0 is default value int type